tensorboard/webapp/widgets/card_fob/card_fob_controller_test.ts

```diff
@@ -207,6 +207,46 @@ describe('card_fob_controller', () => {
         mouseUpListener
       );
     });
+
+    it('swaps fobs being dragged when one fobs collide', () => {
```

Comment:
I would like to add two more tests. 

1. A test that drags the end fob past the start fob
2. A test that starts with start.step === end.step then drags the end step down.